etc/zope.conf

```diff
@@ -0,0 +1,1135 @@
+#
+# Default: CLIENT_HOME/Z2.pid
+#
+# Example:
+#
+#    pid-filename /home/chrism/projects/sessions/var/Z2.pid
+
+
+# Directive: lock-filename
+#
+# Description:
+#     The path to a "lock file" which will be locked by Zope while it's
+#     running.  This file is used by zopectl.py to determine if Zope is
+#     currently running.  This defaults to CLIENT_HOME/Z2.lock.
+#
+# Default: CLIENT_HOME/Z2.lock
+#
+# Example:
+#
+#    lock-filename /home/chrism/projects/sessions/var/Z2.lock
+
+
+# Directive: mime-types
+#
+# Description:
+#     Tells Zope about additional mime.types files that should be
+#     loaded.  The files have the same format as the mime.types file
+#     distributed with Apache.  The "mime-types" setting may be given
+#     more than once in the configuration file.
+#
+# Example:
+#
+#     mime-types  $INSTANCE/etc/mime.types
+
+
+# Directive: structured-text-header-level
+#
+# Description:
+#     Set the default starting HTML header level for structured text
+#     documents. The default is 3, which implies that top-level headers
+#     will be created with an <H3> tag.
+#
+# Default: 3
+#
+# Example:
+#
+#    structured-text-header-level 1
+
+# Directive: rest-input-encoding
+#
+# Description:
+#    Specifies the input encoding of re-StructuredText documents
+#    (e.g. 'utf-8', 'iso-8859-15' or any other valid encoding recognized
+#    by Python).  The default is your Python's default encoding.
+#
+# Default: unset (uses system default)
+#
+# Example:
+#
+#    rest-input-encoding iso-8859-15
+
+# Directive: rest-output-encoding
+#
+# Description:
+#    Specifies the output encoding of re-StructuredText documents
+#    (e.g. 'utf-8', 'iso-8859-15' or any other valid encoding recognized
+#    by Python).  The default is your Python's default encoding.
+#
+# Default: unset (uses system default)
+#
+# Example:
+#
+#    rest-output-encoding iso-8859-15
+
+# Directive: rest-header-level
+#
+# Description:
+#     Set the default starting HTML header level for restructured text
+#     documents. The default is 3, which implies that top-level headers
+#     will be created with an <H3> tag.
+#
+# Default: 3
+#
+# Example:
+#
+#    rest-header-level 2
+
+# Directive: rest-language-code
+#
+# Description:
+#    Language code used for some internal translations inside of the docutils
+#    package and for DTD bibliographic elements mapping. See
+#    lib/python/docutils/languages/ for a list of supported language codes.
+#
+# Default: en
+#
+# Example:
+#
+#    rest-language-code de
+
+# Directive: cgi-environment
+#
+# Description:
+#     A section which allows a user to define arbitrary key-value pairs for
+#     use as the initial CGI environment variables. This is useful
+#     when you want to proxy requests from another web server to Zserver,
+#     and would like Zserver's CGI environment to reflect the CGI
+#     environment of the other web server.
+#
+# Default: unset
+#
+# Example:
+#
+# <cgi-environment>
```

Comment:
This is the default file in the image. When running in control center it gets overwritten by the service definition.  This is used during install and if you want to run zope in the image outside of control center.